data/moves.js

```diff
@@ -1977,6 +1977,7 @@ let BattleMovedex = {
 		onTryMove: function (pokemon, target, move) {
 			if (pokemon.hasType('Fire')) return;
 			this.add('-fail', pokemon, 'move: Burn Up');
+			this.hideLastTarget();
```

Comment:
I actually think instead of adding a new `hideLastTarget` function, it would make more sense to hardcode `attrLastMove('[still]')` to remove the target. It's literally always an information leak, after all.